tests/client.rkt

```diff
@@ -79,11 +80,20 @@
   (cond [(not params) req]
         [else (hash-set req 'params params)]))
 
+(define/contract (make-response request result)
```

Comment:
Perhaps we could consider renaming it to `make-expected-response` as `make-response` is not the responsibility of the client.